pages/TheLearnerView.html

```diff
@@ -65,7 +65,7 @@ <h3>
     When the learner finishes playing the exploration, they are given the
     option to rate it:
   </p>
-  <img src="static/images/explorationRating.png" width="500">
+  <img src="static/images/explorationRating.png" width="500" alt="A white dialogue reading,'Congratulations, you have finished!', followed by a panel of 5 stars, out of which 3 are highlighted in yellow indicating a rating of 3/5.">
```

Comment:
Drop the comma before the quotation mark, and replace it with a space.

", followed" --> "This is followed"

", out of which 3" --> ". 3 of these stars"